README.md

```diff
@@ -1,8 +1,57 @@
 # sPoX
 
-## Demo flow
+`sPoX` is a binary that monitors the Bitcoin blockchain for sBTC deposits made to a set of addresses, and when found
+informs Emily about them so the sBTC signers can then process them.
 
-sPox can be tested with the sBTC devenv:
+Its primary usage is to enable automatic bridging of Stacks POX payments to sBTC.
+
+## Building
+
+To build `sPoX`, run:
+```bash
+cargo build --bin spox --release --locked
+```
+
+The binary will be built in `target/release/spox`.
+
+## Configuration
+
+You can specify which deposits to look for and the endpoints to use in a toml file.
+See `src/config/default.toml` for a config starting point. Note that the Stacks configuration is required only to run
+the `get-signers-xonly-key` command.
+
+### Get signers xonly public key
+
+When configuring a deposit, you must specify for what signers public key it is for in `signers_xonly`. This key changes
+over time after sBTC key rotations. To fetch the current key, fill the `stacks` stanza with the Stacks endpoint and 
+deployer address (for Stacks mainnet, see https://github.com/stacks-sbtc/sbtc/blob/main/docker/mainnet/sbtc-signer/signer-config.toml.in#L109).
+
+Then you can run:
+```bash
+./spox -c <config file> get-signers-xonly-key
+```
+to get the latest key from the sBTC registry smart contract.
```

Comment:
I always find it helpful to mention whether the path here can be relative, absolute, or both.